dataland-frontend/tests/e2e/specs/company-metadata/CompanyUpload.ts

```diff
@@ -1,102 +1,61 @@
   {
     executionEnvironments: ["developmentLocal", "development", "development_2"],
     dataEnvironments: ["fakeFixtures"],
   },
   () => {
-    beforeEach(() => {
-      cy.ensureLoggedIn(uploader_name, uploader_pw);
-    });
-
-    it("Check if post company button is disabled if no values are inserted into the upload form", () => {
-      cy.visitAndCheckAppMount("/companies/upload");
-      cy.get('button[name="postCompanyData"]').should("be.disabled");
-    });
-
-    function uploadEuTaxonomyDataForNonFinancials(companyId: string): void {
-      cy.visitAndCheckAppMount(`/companies/${companyId}/frameworks/eutaxonomy-non-financials/upload`);
-      cy.get('button[name="postEUData"]').should("be.visible");
-      cy.get('input[id="reportingObligation-option-yes"][value=Yes]').check({
-        force: true,
-      });
-      cy.get('select[name="assurance"]').select("None");
-      for (const argument of ["capex", "opex"]) {
-        cy.get(`div[title=${argument}] input`).each(($element, index) => {
-          const inputNumber = 10 * index + 7;
-          cy.wrap($element).type(inputNumber.toString(), { force: true });
+    it(
+      "Check if post company button is disabled before filling the upload form, then fill and submit the form to " +
+        "upload a company and assure that it can be accessed via the company view page",
+      () => {
+        cy.ensureLoggedIn(uploader_name, uploader_pw);
+        const companyName = "Test company XX";
+        uploadCompanyViaFormAndGetId(companyName).then((companyId) => {
+          cy.visitAndCheckAppMount(`/companies/${companyId}`);
+          cy.get("body").should("contain", companyName);
         });
       }
-      cy.get("div[title=revenue] input[name=alignedPercentage]").type("0");
-      cy.get("div[title=revenue] input[name=eligiblePercentage]").type("0");
-      cy.get('button[name="postEUData"]').should("not.be.disabled");
-      cy.get('button[name="postEUData"]').click({ force: true });
-    }
-
-    it("Upload a company by filling the upload form and assure that it can be accessed via the view company page", () => {
-      const companyName = "Test company XX";
-      uploadCompanyViaFormAndGetId(companyName).then((id) => {
-        cy.visitAndCheckAppMount(`/companies/${id}`);
-        cy.get("body").should("contain", companyName);
-      });
-    });
+    );
 
-    it("Log in as data reader, fill the company upload form and assure that upload fails because of insufficient rights", () => {
-      cy.ensureLoggedIn();
-      const companyName = "Test company";
-      cy.visitAndCheckAppMount("/companies/upload");
-      fillCompanyUploadFields(companyName);
-      cy.get('button[name="postCompanyData"]').click();
-      cy.get("body").should("contain", "Sorry");
-    });
-
-    it("Upload EU Taxonomy Dataset and assure that it can be viewed on the framework data view page", () => {
```

Comment:
It is included in the following test now:

"Log in as data reader, fill the company upload form and the Eu Taxonomy data upload form and assure that " +
        "both upload form submits fail because of insufficient rights",